doc/release-notes.md

```diff
@@ -35,24 +35,29 @@ reindex or re-sync the whole chain.
 
 # Notable changes
 
-## Masternode fix
+## Build changes
 
-The memory usage during the old quorum data cleanup mechanism was reduced.
+The miner has been disabled for Windows binaries built via Guix. This resolves
+an issue with false positive alerts from Microsoft Defender and other antivirus providers
+that were preventing users from installing v20.0.3.
 
-## Wallet fix
+Use a self-signed windows code signing certificate instead of expired one. The
+expired certificate being used previously caused most antivirus software to trigger.
+The new self-signed certificate results in a greatly improved installation experience.
```

Comment:
```suggestion
Use a self-signed Windows code signing certificate instead of expired one. The
expired certificate being used previously caused most antivirus software to trigger.
The new self-signed certificate results in a greatly improved installation experience.
```